Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.9
+import PackageDescription
+
+let package = Package(
+    name: "CapacitorCommunityInAppReview",
+    platforms: [.iOS(.v14)],
+    products: [
+        .library(
+            name: "CapacitorCommunityInAppReview",
+            targets: ["InAppReviewPlugin"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "6.0.0")
```

Comment:
```suggestion
        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
```
The plugin has been updated to support Capacitor 7, so the version should be `7.0.0`